Multisig/UI/App/NavigationRouter.swift

```diff
@@ -173,7 +173,7 @@ class ExtendedNavigationRouter: NavigationRouter {
                 chainId: safeAddress.chainId
             )
             return route
-        case "/transactions/queue":
+        case "/transactions/queue", "/transactions", "/transactions/messages":
```

Comment:
we don't have messages in the app, why supporting this route and forwarding it to tx list?

Reply:
I'm forwarding to the closest possible screen, thus handling that specific URL as well.

Reply:
I think there is no need to forward something that is not available in the app

Reply:
can add the explanatory comment, but todo is too much now, as implementation of this is not planned yet